.github/workflows/test.yml

```
@@ -30,11 +30,16 @@ jobs:
         run: npm run bootstrap
       - name: Run tests
         run: npm test
-# Smoke tests disabled while new homepage layout is worked on for Core
-#      - name: Build for smoke tests
-#        run: npm run app-build:smoke
-#      - name: Run smoke tests
-#        timeout-minutes: 10 # sometimes jest fails to exit - https://github.com/facebook/jest/issues/6423#issuecomment-620407580
-#        run: npm run test:smoke:build
+      - name: Build for smoke tests
+        run: npm run app-build:smoke
+      - name: Run smoke tests
+        timeout-minutes: 10 # sometimes jest fails to exit - https://github.com/facebook/jest/issues/6423#issuecomment-620407580
+        run: npm run test:smoke:build
+      - name: Upload smoke test screenshots
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
```

Probably makes sense to mute the warnings for no files found, since that's currently expected
```suggestion
        with:
          if-no-files-found: ignore
```